codecov.yml

```diff
@@ -0,0 +1,29 @@
+comment:
+  layout: "diff, files, header, footer"
+  behavior: default
+
+coverage:
+  # The value range where you want the value to be green
+  # Hold ourselves to a high bar.
+  range: 90..100
+  status:
+    project:
+      coverage:
+        # Use the coverage from the base commit (pull request base) coverage to compare against.
+        # Once we have a baseline we can be more strict.
+        target: auto
+        threshold: 2%
+    patch:
+      default:
+        # Require new code to have 90%+ coverage.
+        target: 90%
+        threshold: 2%
+
+ignore:
+  - "**/tests/**"
+  - "docs/"
+  - "scripts/"
```

Comment:
Should we ignore the `test_common` package? It only includes test helpers

Reply:
Yes, good idea. Just added it to the ignore list